packages/gestalt/src/Box.js

```diff
@@ -301,8 +301,8 @@ const display = value => {
     case false:
       return fromClassName('DisplayNone');
     default:
-      /* block */
-      return fromClassName('DisplayBlock');
+      /* flex */
+      return fromClassName('DisplayFlex', 'DirectionRow');
```

Comment:
If we're here doing this - it would be really awesome to separate these two properties because I'm not sure the horizontal direction makes sense as a default. Could the codemod take

```jsx
<Box display="flex" />
// and convert it to
<Box direction="row" />
```

This might be more intuitive as a default.

Reply:
Thanks! Done